client/src/pages/SavedResources.jsx

```diff
@@ -61,9 +61,10 @@ function SavedResources() {
     setPageSize(parseInt(pageItems, 10));
   }, []);
 
-  if (authed === false) {
+  if (!authed) {
```

Comment:
What does `!authed` return if `authed` is `null`? I'm wondering if we explicitly check for false because we are looking for a non null value

Reply:
This was actually a cause of flaky test - it will return true, which is the behavior we expect (otherwise no redirect)